site/faq.md

```diff
@@ -117,7 +118,7 @@ There are exceptions:
 
 To work around the exceptional cases, you can mount a docker config into
 the Flux container. See the argument `--docker-config` in
-[the daemon arguments reference](https://github.com/weaveworks/flux/blob/master/site/daemon.md).
+[the daemon arguments reference](https://github.com/weaveworks/flux/blob/master/site/daemon.md#flags).
```

Comment:
I think I had just used a relative link here, but no biggie. Let's get this landed.